### ISSUE TITLE
Regarding inputs to trajectory minicking network : Reference motion missing ?

### ISSUE THREAD

**Issue by chinmaySwami** [lsw9021/MASS#36]
Hi,

I was looking into the code and realized that GetState() outputs COM position, velocity and gait phase. This is then used as input during NN training in main.py. 

However, Fig. 5 in the paper indicates reference motion as input, which is not the case in the code.

![image](https://user-images.githubusercontent.com/14370609/230145878-a084a7b3-7fe8-4460-b9b7-a6d00d9baabc.png)

Is it the case that the figure does not represent actual inputs to the network, instead it just represents the components used in the trajectory mimicking phase (reference motion is integral part of reward function) ? 

Can you please help clarify ? 

Thanks,
Chinmay Swami

**Comment by todayThursday** [comment]
I have been training the running model recently, but no matter how to modify reward_param, the training effect is not good, I want to know if you have solved this problem, but on the other hand, I think the weight parameter of the reward function does not affect the effect of the training model, in the code, The reward function only considers the position error and the velocity error of the joint, and scales these two with the error of the end-effector. Logically, if we only consider the similarity of the motion, and do not need to add other effects, just the results of the training of the neural network, then why does the run model not have good results no matter how it is felt
